scripts/complete_liquidity_provision.js

```diff
@@ -122,13 +122,17 @@ module.exports = async (callback) => {
       console.log("==> Skipping safe deployment and using brackets safeOwners")
       bracketAddresses = argv.brackets
       // Ensure that safes are all owned solely by masterSafe
-      await Promise.all(
+      const masterNotOnlyOwner = await Promise.all(
         bracketAddresses.map(async (safeAddr) => {
-          if (!(await isOnlySafeOwner(masterSafe.address, safeAddr))) {
-            callback(`Error: Bracket ${safeAddr} is not owned (or at least not solely) by master safe ${masterSafe.address}`)
-          }
+          !(await isOnlySafeOwner(masterSafe.address, safeAddr))
```

Comment:
For this to work you need to return this value or better to remove the curly brackets, otherwise `masterNotOnlyOwner` is an array of `undefined`.

Reply:
```suggestion
          return !(await isOnlySafeOwner(masterSafe.address, safeAddr))
```